helm_helpers_test.go

```diff
@@ -334,7 +334,7 @@ func Test_getReleaseChartVersion(t *testing.T) {
 		t.Run(tt.name, func(t *testing.T) {
 			t.Log(tt.want)
 			if got := getReleaseChartVersion(tt.args.r); got != tt.want {
-				t.Errorf("getReleaseChartName() = %v, want %v", got, tt.want)
+				t.Errorf("getReleaseChartVersion() = %v, want %v", got, tt.want)
```

Comment:
Does this change have anything to do with windows ? Is it because the test did not report the correct behavior ?

Reply:
I think the old code is wrong. Would you agree? Especially when debugging a unit test, it says something that is not tested here. Here "getReleaseChartVersion" and now it says "getReleaseChartVersion" failed if it fails.